Classes/Private/Manager/Submanagers/OCTSubmanagerCalls.m

```diff
@@ -87,11 +99,44 @@ - (OCTCall *)callToChat:(OCTChat *)chat enableAudio:(BOOL)enableAudio enableVide
     return nil;
 }
 
+- (BOOL)enableVideoSending:(BOOL)enable forCall:(OCTCall *)call error:(NSError **)error
+{
+    OCTToxAVVideoBitRate bitrate = (enable) ? kDefaultVideoBitRate : kOCTToxAVVideoBitRateDisable;
+    BOOL success = [self setVideoBitrate:bitrate forCall:call error:error];
```

Comment:
Is there any need to set video bitrate to 0 when video is disabled? You are stopping videoEngine anyway